### DISCUSSION TITLE
If you want help Mee .

### DISCUSSION THREAD

**Discussion by Youaymo** [termux/termux-packages#11283] [Q&A]
CANNOT LINK EXECUTABLE "apt": cannot locate symbol "__emutls_get_address" referenced by "/data/data/com.termux/files/usr/lib/libapt-pkg.so"

**Comment by Youaymo** [comment]
How to fix this sss

**Comment by 2096779623** [comment]
`termux-info` plz.

https://github.com/termux/termux-packages/issues/10922

**Reply by Youaymo** [reply]
Ok

**Reply by Youaymo** [reply]
$ termux-info
find: ‘/data/data/com.termux/files/usr/etc/apt/sources.list.d’: No such file or directory

**Reply by 2096779623** [reply]
`mkdir -p $PREFIX/etc/apt/sources.list.d/` first.
The newer version has fixed this.

**Comment by licy183** [comment]
Your APP seems not to be a new version. If possible, you could try to download the APP from GitHub Release or F-Droid. Or try https://github.com/termux/termux-packages/issues/10922#issuecomment-1149865896.
